src/Microsoft.DocAsCode.HtmlToPdf/PdfOptions.cs

```diff
@@ -48,6 +48,11 @@ public class PdfOptions
         /// </summary>
         public OutlineOption OutlineOption { get; set; } = OutlineOption.DefaultOutline;
 
+        /// <summary>
```

Comment:
After second thoughts, I think the default value of `TocTitle` should be "Table of Contents".

The consideration under my former suggestion is that I do not want users to find there first page changes from "Cover Page" to "Table of Contents" after upgrading DocFX. However, I think this is minor and acceptable, and it can make cover page scenario more smooth. Sorry for this change.

Reply:
It's not a problem. Thanks! :)